pipes/WDL/workflows/demux_metag.wdl

```diff
@@ -39,6 +47,6 @@ workflow demux_metag {
   }
   call metagenomics.kaiju as kaiju {
     input:
```

Comment:
We haven't really used kaiju regularly via WDL yet, but I'm betting that we may want to consider moving it to a scatter-on-single-sample execution mode (like everything else in our WDLs except kraken). Its database is about 4x smaller (I'm guessing the localization time is just a few minutes) and the execution time of the algorithm is much slower, so the cost efficiency (algorithmic compute time vs VM wall clock time) of kaiju on a single sample is much better than kraken... so we might as well move it within the same scatter block as well.